fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -114,6 +117,8 @@
 
     private final DruidFilterBuilder filterBuilder;
 
+    private Optional<OrderByColumn> dateTimeSort;
```

Comment:
It looks like we're setting it in the constructors to me. The only place I think we're _not_ is in a copy-constructor, to which we should probably add it as a parameter. Once that's done, it _should_ be able to be `final` :-)